cvat-ui/src/components/file-manager/remote-browser.tsx

```diff
@@ -171,6 +171,18 @@ function RemoteBrowser(props: Props): JSX.Element {
             }
         };
         collectNodes(content);
+
+        if (manifestPath) {
+            nodes.push({
+                name: manifestPath,
+                key: manifestPath,
+                children: [],
+                type: 'REG',
+                mimeType: 'unknown',
+                initialized: true,
+            });
+        }
+
         onSelectFiles(nodes);
     }, [selectedRowKeys]);
 
```

Comment:
```suggestion

        onSelectFiles(nodes);
    }, [selectedRowKeys, manifestPath]);

```

Reply:
@Marishka17

If we use some props in useEffect(), we must put these props to useEffect dependencies.